tutorials/6 Analysis/Analyzing a transmon and a resonator.ipynb

```diff
@@ -0,0 +1,654 @@
+    "ℰ_mag = eprd.calc_energy_magnetic()\n",
+    "\n",
+    "print(f\"\"\"\n",
+    "ℰ_elec_all       = {ℰ_elec}\n",
+    "ℰ_elec_substrate = {ℰ_elec_substrate}\n",
+    "EPR of substrate = {ℰ_elec_substrate / ℰ_elec * 100 :.1f}%\n",
+    "\n",
+    "ℰ_mag_all       = {ℰ_mag}\n",
+    "ℰ_mag % of ℰ_elec_all  = {ℰ_mag / ℰ_elec * 100 :.1f}%\n",
+    "\"\"\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "neutral-speaker",
+   "metadata": {},
+   "source": [
+    "### Run EPR analysis\n",
+    "\n",
+    "Perform EPR analysis for all modes and variations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "verified-nigeria",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eprd.do_EPR_analysis()\n",
+    "\n",
+    "# 4a. Perform Hamiltonian spectrum post-analysis, building on mw solutions using EPR\n",
+    "epra = epr.QuantumAnalysis(eprd.data_filename)\n",
+    "epra.analyze_all_variations(cos_trunc = 8, fock_trunc = 7)\n",
+    "\n",
+    "# 4b. Report solved results\n",
+    "swp_variable = 'Lj' # suppose we swept an optimetric analysis vs. inductance Lj_alice\n",
+    "epra.plot_hamiltonian_results(swp_variable=swp_variable)\n",
+    "epra.report_results(swp_variable=swp_variable, numeric=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "mature-batch",
+   "metadata": {},
+   "source": [
+    "Release Ansys session"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "outstanding-museum",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hfss.disconnect_ansys()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "unsigned-lying",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%metal_print Q3D Analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "smart-juice",
+   "metadata": {},
+   "source": [
+    "Next we will look at a lumped oscillator model (LOM) for the same design.\n",
+    "\n",
+    "## Render to Ansys Q3D"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "imported-america",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q3d = design.renderers.q3d"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "environmental-filing",
+   "metadata": {},
+   "source": [
+    "Again, we establish a connection with Ansys and add design for analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "horizontal-updating",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q3d.connect_ansys()\n",
+    "q3d.activate_q3d_design(\"TransmonResonator_q3d\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "caroline-belle",
+   "metadata": {},
+   "source": [
+    "Next, we render the exisitng design to Ansys Q3D for analysis. To ensure that the readout is insulated from the ground plane, we specify the 'open' attribute of render_design to be 'readout'."
```

Comment:
"... we specify the 'open' attribute of render_design to be 'readout'."  ->  "... we set the `readout` pin of Q1  to have an open termination."